app/menus/DocumentMenu.js

```diff
@@ -243,6 +309,11 @@ function DocumentMenu({
               }),
               visible: !!can.createChildDocument,
             },
+            {
+              title: t("Import document"),
```

Comment:
I feel like I want to clarify this menu item but can't quite figure out the wording, perhaps

"Import nested document"

cc @thenanyu

Reply:
Leaving it how it is for now